x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/sources_logic.ts

```diff
@@ -144,11 +144,17 @@ export const SourcesLogic = kea<MakeLogicType<ISourcesValues, ISourcesActions>>(
   selectors: ({ selectors }) => ({
     availableSources: [
       () => [selectors.sourceData],
-      (sourceData: SourceDataItem[]) => sourceData.filter(({ configured }) => !configured),
+      (sourceData: SourceDataItem[]) =>
+        sourceData
+          .filter(({ configured }) => !configured)
+          .sort((a, b) => a.name.localeCompare(b.name)),
     ],
     configuredSources: [
       () => [selectors.sourceData],
-      (sourceData: SourceDataItem[]) => sourceData.filter(({ configured }) => configured),
+      (sourceData: SourceDataItem[]) =>
+        sourceData
+          .filter(({ configured }) => configured)
+          .sort((a, b) => a.name.localeCompare(b.name)),
```

Comment:
AFAIK localeCompare is case sensitive by default. Is that the desired result?

Reply:
localeCompare is case-insensitive afaik: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

Reply:
@sphilipse actually, the documentation page confirms that it's case sensitive, unless an additional parameter is specified.

Reply:
I don't see that in the documentation. Its features depend on the way your locale's language works. For English, it's case-insensitive (in the sense that 'A' < 'b' and 'B' > 'a'). You can see that if you run this in your browser console:
```
const unsortedArray = ['apple', 'Bzzz', 'beer', 'Aaaa'];
unsortedArray.sort((a,b) => a.localeCompare(b));
```
This should result in `['Aaaa', 'apple', 'beer', 'Bzzz']`

There is a somewhat confusingly phrased note example of case-insensitive sorting of an array where they use an unnecessarily complicated example with an ignorePunctuation option, maybe that threw you off?